appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/DesignToolbar.java

```diff
@@ -369,6 +375,22 @@ public void toggleEditor(boolean blocks) {
     setButtonEnabled(WIDGET_NAME_REMOVEFORM, notOnScreen1);
   }
 
+  public void toggleView() {
+    SwitchToBlocksEditorAction blockView = new SwitchToBlocksEditorAction();
+    SwitchToFormEditorAction designView = new SwitchToFormEditorAction();
+    RootPanel.get().addDomHandler(new KeyDownHandler() {
+      public void onKeyDown(KeyDownEvent event) {
+        if (event.isControlKeyDown() && event.isAltKeyDown()) {
```

Comment:
Is there supposed to be a check for a specific key here? This just checks for Ctrl+Alt, which doesn't seem like a safe combination for switching the views.

Reply:
> Is there supposed to be a check for a specific key here? This just checks for Ctrl+Alt, which doesn't seem like a safe combination for switching the views.

It does work, though. Ctrl-Opt on Mac.